sysboundary/ionosphere.cpp

```diff
@@ -1207,13 +1361,33 @@ namespace SBC {
       TracingFieldFunction& BFieldFunction,
       bool outwards
    ) {
+      bool reTrace;
+      uint32_t attempts=0;
       switch(method) {
          case Euler:
             eulerStep(x, v,stepsize, BFieldFunction, outwards);
             break;
+         case ADPT_Euler:
+            do{
+               reTrace=!adaptiveEulerStep(x, v, stepsize, maxStepsize, BFieldFunction, outwards);
+               attempts+=1;
+            }while(reTrace && attempts<= Ionosphere::max_dormand_prince_attempts);
```

Comment:
This is a bit dirty: The adaptive euler now uses max_dormand_prince_attempts. Should that variable rather be "max_tracer_attempts"?

Reply:
Ah right. I wanted to change that to tracer_max_attempts but forgot to. Will do